colibre/scripts/bh_luminosities.py

```diff
@@ -0,0 +1,172 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.stats as stats
+
+from swiftsimio import load
+
+import unyt
+from unyt import mh, cm, Gyr, speed_of_light
+from matplotlib.colors import LogNorm
+from matplotlib.animation import FuncAnimation
+from matplotlib import rcParams
+
+# Set the limits of the figure.
+mass_bounds = [1e5, 3e10]
+value_bounds = [1e38, 1e48]
```

Comment:
```suggestion
mass_bounds = [1e5, 3e10]  # Msun
value_bounds = [1e38, 1e48]  # erg/s
```
Otherwise not clear what the units are

Reply:
True, I have added appropriate comments to all of the scripts.